### PR TITLE
Added Setting Subpages 

### PR DESCRIPTION
**Pull request by pascuad21**
**Purpose:**
Hardcoded the Notification and Profile pages for the settings. 

**Changes**
Created ProfileStackScreen.jsx
Created NotificationsStackScreen.jsx
screens.js
Settings.jsx
globalStyles.js

**Testing**
Tested on Android 

image
image

### CODE REVIEW

File: src/components/NotificationsStackScreen.jsx [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+                /* Condiditonal icon if the notification has been seen */
+                    newCheck(props.new)
+                }
+            </View>
+        </View>
+    );
+}
+
+function newCheck(val){
+    if(val == true){
```
Comment (brmscheiner) [inline src/components/NotificationsStackScreen.jsx:46]: A few things here!
1. `if (val)` will do the same thing
2. avoid using `==` in JS. it's a safer habit to use `===` unless you want to memorize the difference between them and think about it all the time =)
3. make sure your code is linted (eslint, prettier)

Comment (pascuad21) [reply src/components/NotificationsStackScreen.jsx]: Ok I fixed it

**Comment by degrood21** [comment]
Overlooked changes Dylan made in response to Ben. If(val) is implemented and files are linted

**Review by degrood21 — APPROVED** [review]
Overlooked changes Dylan made in response to Ben. If(val) is implemented and files are linted

**Pull request merged**